assignment03/script.js

```diff
@@ -0,0 +1,58 @@
+// 1. Declare a variable let age = 25;. Write a series of if else statements that will:
+//     Print child to the console if age is less than equal to 12.
+//     Print teen to the console if age is between 13 and 18(inclusive).
+//     Print adult to the console if age is above 18
+
+let age = 25
+
+// if-else statements
+
+if(age<=12) {
+    console.log("child")
+}
+else if(age<=18){
+    console.log("teen");
+}
+else 
+    console.log('adult');
+
+// 2.  Do the same using switch case.
+// Switch statements
+
+switch(age){
```

Comment:
This switch case won't work. It would print adult always or most of the time

Reply:
Updated switch case